### ISSUE TITLE
[ FEATURE ] Latest API endpoint

### ISSUE THREAD

**Issue by FlavioAdamo** [Exifly/ApiVault#259]
## Is your feature request related to a problem? Please describe
I have noticed that our current API system lacks the capability to retrieve the latest APIs that have been added to our platform. This presents a significant challenge for users who want to stay up-to-date with the latest additions. 

## Describe the solution you'd like
I would like to request the implementation of a new endpoint in our API system that allows users to retrieve the latest APIs added to our platform. This endpoint should return a list of APIs sorted by their addition date, with the most recent ones appearing first. To facilitate this, we will also need to modify our API model to include a 'date' field that tracks when each API was added to the platform.

## Additional context
Adding this feature will greatly enhance the user experience and make it easier for developers and other users to discover and integrate the latest APIs into their projects. It will also improve the overall usability of our platform by providing a clear and concise way to access up-to-date API information. 

Branch: feat_feedback_panel

**Comment by gdjohn4s** [comment]
Linked feature branch to the issue

**Comment by zackhamr** [comment]
Hey, i am gonna take a crack at this unless someone has already start on it. 

This is my first time contributing to an open source project so any pointers will be greatly appreciated.

**Comment by FlavioAdamo** [comment]
Sure, Issue assigned!
If you need any help feel free to reach me :)

**Comment by gdjohn4s** [comment]
@zackhamr Hi, any news about the issue?

**Comment by zackhamr** [comment]
gonna spend some time on it later today after I finish a couple applications

**Comment by gdjohn4s** [comment]
no problem 😄

**Comment by zackhamr** [comment]
So im having issues with docker (my first time using it), the error I get when I run compose is 

`Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:9001 -> 0.0.0.0:0: listen tcp 0.0.0.0:9001: bind: An attempt was made to access a socket in a way forbidden by its access permissions.`

and looking online I found that tends to mean that port is being used.

So I tried changing the port and began getting issue with changing postrges permissions
`changing permissions of '/var/lib/postgresql/data/pgdata/': Operation not permitted`

which I believe is the cause of some other warnings am i getting 
` django.db.utils.OperationalError: could not translate host name "database" to address: Temporary failure in name resolution`

has anyone else dealt with this or know how to fix this?
any other docker tips would also be helpful. Been going through some youtube tutorials which are helpful but not really for my situation .

**Comment by zackhamr** [comment]
Yeah using the one in ApiVault, I am using windows 10 and using wsl ubuntu terminal.

**Comment by gdjohn4s** [comment]
Mhh have you tried removing the `data-db` directory before restarting your docker-compose?
